schutzbot/get_civ_config.py

```diff
@@ -165,23 +166,23 @@ def get_modified_methods_str():
     return " or ".join(list(modified_methods))
 
 
-def get_skip_vars():
-    skip_vars = {
+def get_skip_vars() -> Dict[str, str]:
+    skip_vars: Dict[str, str] = {
         "skip_aws": "false",
         "skip_azure": "false",
         "skip_gcp": "false"
     }
 
-    files_changed = get_files_changed()
+    files_changed: List[str] = get_files_changed()
 
     for file in files_changed:
         if "test_suite/generic/.*" in file:
             return skip_vars
 
     return {
-        "skip_aws": "test_suite/cloud/test_aws.py" in files_changed,
-        "skip_azure": "test_suite/cloud/test_azure.py" in files_changed,
-        "skip_gcp": "test_suite/cloud/test_gcp.py" in files_changed,
+        "skip_aws": "false" if "test_suite/cloud/test_aws.py" in files_changed else "true",
```

Comment:
Oh dear, now I see what happened. I was storing booleans and with the inverted logic here, instead of strings.

We need to rework this and simplify things...

Reply:
Would a revert be better in this case?
At this point I'm not sure if I broke things up due to my refactoring :/.
In the reverted logic, we can just nest some things inside an `if` statement.
The code will still be hard to maintain though (maybe harder than in this refactored version).